vendor/k8s.io/system-validators/validators/types_unix.go

```diff
@@ -57,6 +57,16 @@ var DefaultSysSpec = SysSpec{
 		Forbidden: []KernelConfig{},
 	},
 	Cgroups: []string{"cpu", "cpuacct", "cpuset", "devices", "freezer", "memory"},
+	CgroupsOptional: []string{
+		// The hugetlb cgroup is optional since some kernels are compiled without support for huge pages
+		// and therefore lacks corresponding hugetlb cgroup
+		"hugetlb",
+		// The pids cgroup is optional since it is only used when at least one of the feature flags "SupportPodPidsLimit" and
+		// "SupportNodePidsLimit" is enabled
+		"pids",
+	},
+	CgroupsV2:         []string{"cpu", "cpuset", "devices", "freezer", "memory"},
```

Comment:
cpuacct is intentionally excluded?

Reply:
yes https://github.com/kubernetes/system-validators/pull/12#pullrequestreview-385627154